src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -493,6 +507,78 @@ public int[] EndTime
             }
         }
 
+        /// <summary>
+        /// Get the list of the properties of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<string> PropertyList
+        {
+            get
+            {
+                if (null == _propertyList)
+                {
+                    _propertyList = new List<string>();
```

Comment:
in c#, pascal case is supposed to be used and for private member, it would be OK to use `_` prefix. However in NUI, we have decided **NOT to use prefix such as `_`, `m`, `p`, and etc.**

- NUI ground rule : https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule
- NUI coding rule : https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=191373542

could you use **PascalCase** even for private fields?